src/Core/Settings.h

```diff
@@ -770,6 +770,7 @@ class IColumn;
     M(Bool, allow_experimental_undrop_table_query, false, "Allow to use undrop query to restore dropped table in a limited time", 0) \
     M(Bool, keeper_map_strict_mode, false, "Enforce additional checks during operations on KeeperMap. E.g. throw an exception on an insert for already existing key", 0) \
     M(UInt64, extract_kvp_max_pairs_per_row, 1000, "Max number pairs that can be produced by extractKeyValuePairs function. Used to safeguard against consuming too much memory.", 0) \
+    M(Timezone, session_timezone, "", "The default timezone for current session or query. The server default timezone if empty.", 0) \
```

Comment:
Hm, there is `use_client_time_zone` (with underscore), maybe it worth to add an alias for `use_client_timezone` as well to comply the convention?

Although it seems that `time_zone` and `timezone` has a more or less the same distribution:

```
 gg time_zone  | wc -l
1069
$ gg timezone  | wc -l
987

$ gg '\<timezone\>'  | wc -l
643
$ gg '\<time_zone\>'  | wc -l
848
```